utilsKinematics.py

```diff
@@ -37,13 +39,18 @@ def __init__(self, dataDir, trialName,
         
         # Model.
         opensim.Logger.setLevelString('error')
-        modelPath = os.path.join(dataDir, 'OpenSimData', 'Model',
+        # check if specified model name exists
+        modelPath = os.path.join(sessionDir, 'OpenSimData', 'Model',
                                  '{}.osim'.format(modelName))
+        if not os.path.exists(modelPath):
```

Comment:
@suhlrich I don't think we want this. Say a user wants to do an analysis with a model they changed but they make a typo in their model name. This will automatically pull the model from the database. What was the use case for this?

Reply:
ok. now if you specify a model, it loads that, if you don't, it loads from server. If the file isn't there, it errors.